### PR TITLE
Support for mongoid 5

### PR DESCRIPTION
**Pull request by nazgum**
Quick update for mongoid 5

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,9 +3,11 @@ source "http://rubygems.org"
 # Specify your gem's dependencies in mongoid_orderable.gemspec
 gemspec
 
-case version = ENV['MONGOID_VERSION'] || "~> 3.1"
+case version = ENV['MONGOID_VERSION'] || "~> 4.0"
+when /5/
+  gem "mongoid", "~> 5.0.0"
```
Comment (pyromaniac) [inline Gemfile]: Why not "~> 5.0" ?

File: Gemfile [hunk 2]
```diff
@@ -3,9 +3,11 @@ source "http://rubygems.org"
 # Specify your gem's dependencies in mongoid_orderable.gemspec
 gemspec
 
-case version = ENV['MONGOID_VERSION'] || "~> 3.1"
+case version = ENV['MONGOID_VERSION'] || "~> 4.0"
```
Comment (pyromaniac) [inline Gemfile]: `|| "~> 5.0"`

File: Gemfile (hunk 1 above)
Comment (nazgum) [inline Gemfile]: Just used the recommended value from the mongoid 5 docs - https://docs.mongodb.org/ecosystem/tutorial/ruby-mongoid-tutorial/ - but aye, to be consistent with your prior versions perhaps should have put ~> 5.0

**Comment by dblock** [comment]
More complete support with all the spec fixes in https://github.com/pyromaniac/mongoid_orderable/pull/41

**Comment by pyromaniac** [comment]
Thanks

**Pull request closed without merging**